package.json

```diff
@@ -12,8 +12,14 @@
     "node": ">= 0.10.x"
   },
   "scripts": {
-    "test": "scripts/test",
-    "prepublish": "scripts/build"
+    "build": "coffee -b -c -o lib/ src/ && cpy 'src/**/*.js' lib",
```

Comment:
@danielgtaylor I'd vote to use `compile`, as it won't create possible problems with `npm rebuild`